app/scripts/BarTrack.js

```diff
@@ -258,7 +266,8 @@ class BarTrack extends HorizontalLine1DPixiTrack {
 
   drawZeroLineSvg(output) {
     const zeroLine = document.createElement('rect');
-    zeroLine.setAttribute('id', 'zero-line');
+    const zeroLineUniqueID = Math.random().toString(36).substring(7);
```

Comment:
Same comment about using `slugid` for generating random ids:

As in here:

https://github.com/higlass/higlass/blob/develop/app/scripts/TrackRenderer.jsx#L272